matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/space/DefaultSpaceService.kt

```diff
@@ -184,6 +184,10 @@ internal class DefaultSpaceService @Inject constructor(
         return joinSpaceTask.execute(JoinSpaceTask.Params(spaceIdOrAlias, reason, viaServers))
     }
 
+    override suspend fun leaveSpace(roomId: String, reason: String?) {
```

Comment:
spaceId instead of roomId?

Reply:
initially changed to spaceId, but since we use Room for Space sometimes wasn't sure if it won't be confusing. Like
leaveSpace(spaceId = room.roomId). What do you think?